src/Item.js

```diff
@@ -1,8 +1,10 @@
 import React, { useState, useEffect } from 'react';
 import { db } from './lib/firebase';
+import estimates from './lib/estimates';
+import { intervalToDuration, fromUnixTime } from 'date-fns';
 
-function Item(props) {
-  const { purchaseDates, userToken, itemName, id } = props;
+function Item({ userToken, item }) {
```

Comment:
Huge fan of the props destructuring! Makes the rest of the code so much cleaner!🔥